src/SFA.DAS.ApplyService.InternalApi/Authentication/AzureAdScopeClaimTransformation.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApplyService.InternalApi.Authentication
+{
+    public class AzureAdScopeClaimTransformation : IClaimsTransformation
+    {
+        public Task<ClaimsPrincipal> TransformAsync(ClaimsPrincipal principal)
+        {
+            var scopeClaims = principal.FindAll(Constants.ScopeClaimType).ToList();
+            if (scopeClaims.Count != 1 || !scopeClaims[0].Value.Contains(' '))
```

Comment:
I think this needs to ben scopeClaims.Count>1 and you might need to cope with scopeClaims.Count==0

wouldn't scopeClaims.First().Value be more readable maybe?

Reply:
Not sure. This was copied directly from a common package (would have used it but not compatible with Apply, sadly)